e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/multiple-contract-policy.spec.js

```diff
@@ -264,4 +285,47 @@ context('Insurance - Policy and exports - Multiple contract policy page - As an
       expect(text.trim()).equal(BUTTONS.SAVE_AND_BACK);
     });
   });
+
+  describe('form submission', () => {
+    const date = new Date();
+    const startDate = add(date, { months: 3 });
+
+    it(`should redirect to ${ABOUT_GOODS_OR_SERVICES}`, () => {
+      cy.completeAndSubmitMultipleContractPolicyForm();
+
+      const expectedUrl = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ABOUT_GOODS_OR_SERVICES}`;
+      cy.url().should('eq', expectedUrl);
+    });
+
+    describe('after submitting the form', () => {
+      it('should retain the `type of policy and exports` task status as `in progress`', () => {
+        cy.visit(`${INSURANCE_ROOT}/${referenceNumber}${ALL_SECTIONS}`, {
+          auth: {
+            username: Cypress.config('basicAuthKey'),
+            password: Cypress.config('basicAuthSecret'),
+          },
+        });
+
+        const task = taskList.prepareApplication.tasks.policyTypeAndExports;
+
+        task.status().invoke('text').then((text) => {
+          const expected = TASKS.STATUS.IN_PROGRESS;
+
+          expect(text.trim()).equal(expected);
+        });
+      });
+    });
+
+    describe('when going back to the page', () => {
+      it('should have the submitted values', () => {
+        goToPageDirectly(referenceNumber);
+
+        multipleContractPolicyPage[REQUESTED_START_DATE].dayInput().should('have.value', '1');
+        multipleContractPolicyPage[REQUESTED_START_DATE].monthInput().should('have.value', getMonth(startDate));
+        multipleContractPolicyPage[REQUESTED_START_DATE].yearInput().should('have.value', getYear(startDate));
+
+        multipleContractPolicyPage[TOTAL_MONTHS_OF_COVER].inputOptionSelected().contains('2');
```

Comment:
is there are reason we are using contains and not equal?

Reply:
The rendered select options have "months" string - didn't see much value in adding this in here. Can add if you'd like?

Reply:
I am just thinking if the months changed to say 23, this test will still pass

Reply:
Good point, i'll update it 👍